### DISCUSSION TITLE
token expiration

### DISCUSSION THREAD

**Discussion by anton-ub** [eclipse-edc/Samples#323] [Q&A]
In example [transfer-02-consumer-pull] consumer obtains in the end url and token to fetch actual json data. Although this token has no expiration date. How would it be possible to set the limit for the access time here?

**Comment by paullatzelsperger** [comment]
you would need to implement your own token expiration - and more importantly: renewal process. EDC intentionally leaves that to the adopters.

**Reply by anton-ub** [reply]
but how to do it?
